setup.cfg

```diff
@@ -45,6 +45,7 @@ install_requires =
     scipy>=1.9.2
     scikit_learn>=1.0.0
     koffi>=0.1.1
+    jplephem
```

Comment:
Is this used anywhere?

Reply:
yes. It is used in _calc_barycentric_corr when a distance is specified. Which it never was until these tests.
line 383
            obs_pos = get_body_barycentric("earth", Time(mjdlist[0], format="mjd"))